slides/02/02.md

```diff
@@ -102,7 +102,7 @@ Note that the $L^2$-regularization is usually not applied to the _bias_, only to
 ~~~
 Also, without penalizing the bias, linear regression with $L^2$-regularization is invariant
 to shifts (i.e., adding a constant to all the targets results in the same solution, only with
-the bias increased by that constant; if the bias was penalized, this would not
+the bias increased by that constant; if the bias were penalized, this would not
```

Comment:
You are the second one to suggest this, so I looked it up and learned how to use "was/were" in if clause of a conditional sentence -- thanks you all, I learned something new ;-)